Fix documentation for `--lock-timeout` (#16021)
vitessio/vitess: go/flags/endtoend/vtbackup.txt

      --keep-alive-timeout duration                                 Wait until timeout elapses after a successful backup before shutting down.
      --keep_logs duration                                          keep logs for this long (using ctime) (zero to keep forever)
      --keep_logs_by_mtime duration                                 keep logs for this long (using mtime) (zero to keep forever)

>       --lock-timeout duration                                       Maximum time to wait for while acquiring a lock from the topo server (default 45s)

Reviewer: Supper nitty, but small suggestion:
```suggestion
      --lock-timeout duration                                       Maximum time to wait when attempting to acquire a lock from the topo server (default 45s)
```
